app/api/routers/breeze_buddy/deprecated/outbound_numbers.py

```
@@ -67,13 +57,12 @@ async def add_outbound_number(
 
 @router.get("/outbound-number")
 async def get_outbound_number(
-    id: str = None, current_user: TokenData = Depends(get_current_user)
+    id: str = None,
```

Ruff (0.14.13)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)